util/gh-pages/script.js

```diff
@@ -138,18 +138,20 @@ function onEachLazy(lazyArray, func) {
     }
 }
 
-function highlightIfNeeded(elem) {
-    onEachLazy(elem.querySelectorAll("pre > code.language-rust:not(.highlighted)"), el => {
+function highlightIfNeeded(lintId) {
+    const elem = document.getElementById(lintId);
+    const call = el => {
         hljs.highlightElement(el.parentElement)
         el.classList.add("highlighted");
-    });
+    };
+    onEachLazy(elem.querySelectorAll("pre > code.language-rust:not(.highlighted)"), call);
+    onEachLazy(elem.querySelectorAll("pre > code.language-no_run:not(.highlighted)"), call);
```

Comment:
Ah this should not be happening, some changes from #13359 may need to be reapplied on top of those for #13269

Reply:
Oh I see. Gonna update then.